rholang/src/test/scala/coop/rchain/rholang/interpreter/SubstituteTest.scala

```diff
@@ -2,37 +2,37 @@ package coop.rchain.rholang.interpreter
 
 import org.scalatest.{FlatSpec, Matchers}
 import Substitute._
-import scala.collection.immutable.HashMap
 
 class VarSubSpec extends FlatSpec with Matchers {
 
   "FreeVar" should "throw an error" in {
-    val env = HashMap(0 -> Par(GPrivate()))
+    val env = Env(Par(GPrivate()))
     an[Error] should be thrownBy subOrDec(FreeVar(0))(env)
   }
 
   "BoundVar" should "be substituted for process" in {
     val source = Par(GPrivate())
-    val env    = HashMap(0 -> source)
+    val env    = Env(source)
     val result = subOrDec(BoundVar(0))(env)
     result should be(Right(source))
   }
 
   "BoundVar" should "be substituted with renamed expression" in {
     val source  = Par(Send(ChanVar(BoundVar(0)), List(Par()), false, 0))
     val _source = Par(Send(ChanVar(BoundVar(1)), List(Par()), false, 0))
-    val env     = HashMap(0 -> source)
+    val env     = Env(source)
     val result  = subOrDec(BoundVar(0))(env)
     result should be(Right(_source))
   }
 
   "BoundVar" should "throw an error" in {
-    val env = HashMap(1 -> Par(GPrivate()))
+    val env = Env[Par](1 -> Par(GPrivate()))
+    println(env)
```

Comment:
Seems like you got some debugging cleanup. Also probably stay consistent on `Env[Par](1 -> Par(GPrivate())` vs `Env(1 -> Par(GPrivate())`

Reply:
Ah. Thank you!